### ISSUE TITLE
Revert Story view 

### ISSUE THREAD

**Issue by Odai773** [c2mInc/C2MStoryViewer#13]
I want to revert story view (the progress bar and and cube out transformation) to be from right to left the code is complex so I want your help to achieve this.

Thank you in advance.

**Comment by Cutta** [comment]
Hi @Odai773 
Have you made some progress on this?

**Comment by Odai773** [comment]
Yes in PausableProgressBar class i changed the parameter of PausableScaleAnimation and it dose not work correctly

**Comment by Odai773** [comment]
@Cutta Do you have some steps for me to follow regarding this issue?

**Comment by Cutta** [comment]
[Issue 4](https://github.com/c2mInc/C2MStoryViewer/issues/4) may help you for now. I will try to make time to deal with

**Comment by Odai773** [comment]
Thank you @Cutta for your time
